frontend/src/locales/en.json

```
@@ -176,7 +202,9 @@
   "login": "Login",
   "math": {
     "aprox": "~",
+    "divide": "/",
     "equal": "=",
+    "equalTo": "<",
```

```suggestion
    "lower": "<",
```

Change all usages as well !!!